plugin/main/resources/GeneratedAndroidProject/microbenchmark/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.jetbrains.kotlin.android)
+}
+
+android {
+    namespace = "kotlinx.benchmark.android.microbenchmark"
+    compileSdk = 34
```

Comment:
When generating a project, compile SDK and runtime SDK versions as well as other properties, like defaultConfig and compileOptions, should be overridden by values inferred from a project the plugin was applied to.